src/functionalTest/resources/features/common/data-store/FT_NoCCaseType/snippets/correct_question_verification.td.json

```diff
@@ -0,0 +1,8 @@
+{
+  "_guid_" : "correct_question_verification",
```

Comment:
Should be moved into MCA common folder (i.e.  `src/functionalTest/resources/features/common/mca`) as it is a snippet of a call to an MCA API not a data-store one.